### DISCUSSION TITLE
Enable profiles with JavaScript Functional Programming

### DISCUSSION THREAD

**Discussion by alex-morgun** [codewars/codewars.com#3121] [Ideas]
It would be great to have option to use popular Functional Programming libraries like lodash and ramda.
I understand that there are no requests for this future, but it will become more popular when people would see simplicity and expressiveness of lodash/ramda.

Enable profiles with core-js + lodash or core-js + ramda

https://lodash.com/
https://ramdajs.com/

**Comment by hobovsky** [comment]
Lodash and Ramda are already available for use in JavaScript kata (see the list of [modules available for JavaScript](https://docs.codewars.com/languages/javascript#node-modules)).

Additionally, the appropriate way to request a new language, language version, or a library, is to create a ticket in [`codewars/runner` repository](https://github.com/codewars/runner/issues). Please create a ticket there, using an appropriate template.

Closing the discussion as off-topic for this area.
